socorro/signature/rules.py

```diff
@@ -525,9 +525,12 @@ def action(self, crash_data, result):
             # If we have a thread to look at, pull the frames for that.
             # Otherwise we don't have frames to use.
             if crashing_thread is not None:
+                crashing_thread_mapping = glom(
+                    crash_data, "threads.%d" % crashing_thread, default={}
+                )
+                crashing_thread_name = crashing_thread_mapping.get("thread_name", None)
                 signature_list = self._create_frame_list(
-                    glom(crash_data, "threads.%d" % crashing_thread, default={}),
-                    crash_data.get("os") == "Windows NT",
+                    crashing_thread_mapping, crash_data.get("os") == "Windows NT"
```

Comment:
I don't understand this. You're not adding the `crashing_thread_name` or `crashing_thread_id` to the signature, so why make this change?